src/components/Summary/index.js

```diff
@@ -8,8 +8,8 @@ class Summary extends Component {
   render() {
     return (
       <div className="Summary">
-        { false ? <Recap /> : '' }
-        <Progress />
+        { this.props.showRecap ? <Recap bath={this.props.bath} /> : '' }
```

Comment:
Use [short-circuit](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators#short-ciurcuit_evaluation) operators when you only want to consider one case